changelogs/current.yaml

```diff
@@ -65,6 +65,10 @@ minor_behavior_changes:
     Added config name of filter sending a local reply in filter state with key
     ``envoy.filters.network.http_connection_manager.local_reply_owner``.
     See :ref:`the well-known filter state keys <well_known_filter_state>` for more detail.
+- area: filter state
+  change: |
+    Added matched action name in composite extension in filter state with key
```

Comment:
Reword a bit?
```suggestion
    Added composite filter's matched action name to filter state with key
```

Reply:
Thanks, texts are difficult to me.

In addition I think we should add some reference here
https://github.com/envoyproxy/envoy/blob/main/docs/root/configuration/http/http_filters/composite_filter.rst
 but I don't know how to do it

Reply:
You can update that file (docs/root/configuration/http/http_filters/composite_filter.rst) directly. You can find some examples in its file history https://github.com/envoyproxy/envoy/commits/main/docs/root/configuration/http/http_filters/composite_filter.rst